app/src/main/java/com/dessalines/thumbkey/utils/Utils.kt

```diff
@@ -261,12 +261,46 @@ fun swipeDirection(
     y: Float,
     minSwipeLength: Int,
     swipeType: SwipeNWay = SwipeNWay.EIGHT_WAY,
+    ghostSwipeType: SwipeNWay? = null,
 ): SwipeDirection? {
     val xD = x.toDouble()
     val yD = y.toDouble()
 
     val swipeLength = sqrt(xD.pow(2) + yD.pow(2))
 
+    val compositeSwipeType =
```

Comment:
This seems a bit overkill. If ghost keys are enabled, its probably simple enough to just make it `EIGHT_WAY`